api/v2/performanceprofile_types.go

```diff
@@ -123,6 +126,36 @@ type NUMA struct {
 	TopologyPolicy *string `json:"topologyPolicy,omitempty"`
 }
 
+// Net defines a set of network related features
+type Net struct {
+	// UserLevelNetworking defines if the specified network devices . Defaults to "false".
+	UserLevelNetworking *bool `json:"userLevelNetworking,omitempty"`
+	// Devices contains a list of network device representations that will be
+	// set with a netqueue count equals to CPU.Reserved .
+	// If no devices are specified then the default is all devices.
+	Devices []Device `json:"devices,omitempty"`
+}
+
+// Device defines a way to represent a network device in several options:
+// device name, vendor ID, model ID, PCI path and MAC address
+type Device struct {
+	// Network device name
+	// +optional
+	Name *string `json:"name,omitempty"`
+	// Network device vendor ID represnted as a 16 bit Hexmadecimal number.
+	// +optional
+	VendorID *string `json:"vendor,omitempty"`
+	// Network device model ID (device) represnted as a 16 bit hexmadecimal number.
+	// +optional
+	ModelID *string `json:"model,omitempty"`
+	// Network device PCI path
+	// +optional
+	PCIpath *string `json:"pciPath,omitempty"`
+	// Network device MAC address
+	// +optional
+	MAC *string `json:"mac,omitempty"`
```

Comment:
Please ensure syntactic validation for all the fields. I would use the standard "ff":"ff": notation.
I do not understand how we can use MAC addresses to identify similar devices on different nodes? Are we going to use some "pattern" based matching? Or is this a way to enumerate all devices on all nodes if we don;t have other ways to identify them?
Other points:
 - There are net devices that have multiple MAC addresses
 - MAC addresses can be changed by drivers / firmware updates and so on AFAIK